dangerfile.js

```diff
@@ -0,0 +1,142 @@
+/* global danger, fail, message */
+import { message, danger } from 'danger';
+
+const modifiedMD = danger.git.modified_files.join('- ');
+message(`Changed Files in this PR: \n - ${modifiedMD}`);
+
+// const { pr } = danger.github;
```

Comment:
Why are we adding all this commented out code?

Reply:
Because it's a hack and I haven't added anything useful to it yet or tidied it up 😜

Reply:
ffs, sorry I thought this was a real PR. Ignore me.